pom.xml

```diff
@@ -271,6 +271,13 @@
 
 	<!-- All children will have these dependencies -->
 	<dependencies>
+            
+                <!--For String Joining, etc. -->
+                <dependency>
+                        <groupId>org.apache.commons</groupId>
```

Comment:
if not ALL modules need this, put it under dependencyManagement please.

Reply:
its already there
removed the version from below

Reply:
then it shouldn't be in `dependencies` as it makes it a dependency on all modules.